pages/learning_MDAnalysis.md

```diff
@@ -4,64 +4,27 @@ title: Learning MDAnalysis
 order: 3
 ---
 
-Once you had a look at the 
-[basic example]({{ site.baseurl }}/pages/basic_example) 
-you might want to learn more about how to use
-MDAnalysis. MDAnalysis is primarily a library that helps you to build
-your own tools but it also works very well for **interactive data
-exploration** of MD data in [IPython](http://ipython.org/), in
-particular within [Jupyter notebooks](https://jupyter.org/) and in
-conjunction with [pandas](http://pandas.pydata.org/). MDAnalysis is
-well suited for a *rapid development* approach.
-
-The resources below should help you to quickly find out to best use
-MDAnalysis for your own specific uses.
+MDAnalysis is a powerful Python library for analyzing MD simulations. While primarily designed to help you build custom analysis tools, it also supports interactive data exploration in environments like [IPython](http://ipython.org/) and [Jupyter notebooks](https://jupyter.org/), especially when combined with [pandas](http://pandas.pydata.org/). This makes MDAnalysis an excellent choice for rapid prototyping and exploratory analysis.
 
+Whether you're new to MDAnalysis or looking to deepen your expertise, this page will guide you through our learning resources.
 
 ## Tutorials ##
 
-Start with the [{{ site.docs.quickstart.name }}]({{
-site.docs.quickstart.url }}) when you are new MDAnalysis.
-
-Then browse the [{{ site.docs.userguide.name }}]({{ site.docs.userguide.url }}), which contains detailed documentation for all the important parts of MDAnalysis and many self-contained tutorials.
-
-
-There are a number of [older tutorials]({{site.github.wiki}}/Tutorials) available, too, although we recommend new users start  with [{{ site.docs.quickstart.name }}]({{
-site.docs.quickstart.url }}) and then start reading the [{{ site.docs.userguide.name }}]({{ site.docs.userguide.url }}).
-
-
-## Documentation ##
+If you are new to MDAnalysis, start with:
 
-The [{{ site.docs.userguide.name }}]({{ site.docs.userguide.url }})
-contains installation instructions, the [{{ site.docs.quickstart.name
-}}]({{ site.docs.quickstart.url }}), and comprehensive description of
-the functionality of MDAnalysis from a user's perspective. **New users
-should start here!**
+- [{{ site.docs.quickstart.name }}]({{ site.docs.quickstart.url }}) – A quick introduction to MDAnalysis.
+- [{{ site.docs.userguide.name }}]({{ site.docs.userguide.url }}) – Detailed documentation with in-depth tutorials.
 
-The [{{ site.docs.mdanalysis.name }}]({{ site.docs.mdanalysis.url }})
-contains technical information on how to use MDAnalysis. 
-
-The [paper on MDAnalysis]({{ site.baseurl
```

Comment:
Can we still mention the papers somewhere? MDAnalysis is academic software so papers _are_ important.

Having papers referenced also indicates that this is "published" research, which may add to the confidence that new users may have in the software.